templates/default/css/template.css

```diff
@@ -622,3 +624,47 @@ div.yes-no .btn-group label {
     border: none;
     }
 }
+
+@media only screen and (max-width: 760px){
+  h1, .h3{
+    font-size:18px;
+  }
+  
+  .h4{
+    font-size:16px;
+  }
```

Comment:
Like you want :) i think i try to fix the bug for array by dual scale (https://bugs.limesurvey.org/view.php?id=11863) before